wren/compute/materialization.go

```diff
@@ -0,0 +1,175 @@
+package compute
+
+import (
+	"context"
+
+	v1alpha "github.com/kaskada-ai/kaskada/gen/proto/go/kaskada/kaskada/v1alpha"
+	"github.com/kaskada-ai/kaskada/wren/customerrors"
+	"github.com/kaskada-ai/kaskada/wren/ent"
+	"github.com/kaskada-ai/kaskada/wren/ent/materialization"
+	"github.com/rs/zerolog/log"
+)
```

Comment:
wrappers around calling the new methods in the compute engine for managing long-lived materializations